src/api/middlewares/RpcMiddleware.ts

```diff
@@ -16,9 +17,8 @@ export class RpcMiddleware implements interfaces.Middleware {
         // validate rpc request
         if (!this.isValidVersionTwoRequest(req)) {
             return res.failed(400, 'Invalid JSON-RPC 2.0 request');
-        } else {
-            next();
         }
+        next();
     }
```

Comment:
here also:
if( isValidVersionTwoRequest() ) return next()
and everything else: fail